examples/build_report.py

```diff
@@ -55,7 +55,7 @@
 build = project.build(args.build)
 
 squad_envs = Squad().environments(project=project.id)
-squad_suites = Squad().suites(project=project.id)
+squad_suites = Squad().suites(project=project.id, count=-1)
```

Comment:
@mwasilew ALL is already imported. I think it would look better if we use it.
it is declared here https://github.com/Linaro/squad-client/blob/3ea98eae12b380b2ddb53a2d6b7585dac1761cb9/squad_client/core/models.py#L15